### ISSUE TITLE
lmlsb extractmenu errored

### ISSUE THREAD

**Issue by shy2052** [pmrowla/pylivemaker#75]
* pylivemaker version: 1.0.0
* Python version: 3.7.8
* Operating System: windows 7
* LiveMaker game: Ikisugi M Lesson

### Description

It only errors on this file.
[00000001.zip](https://github.com/pmrowla/pylivemaker/files/5427177/00000001.zip)

### What I Did

```
E:\GAME\Ikisugi M Lesson\ikiM\game_files>lmlsb extractmenu --encoding=utf-8-sig 00000001.lsb menu.csv
Extracting 00000001.lsb ...
Traceback (most recent call last):
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\livemaker\lsb\menu.py", line 431, in from_lsb_command
    target, target_index = jumps[name]
KeyError: 'スタート'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\Scripts\lmlsb.exe\__main__.py", line 7, in <module>
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\livemaker\cli\lmlsb.py", line 987, in extractmenu
    for id_, choice in lsb.get_menu_choices():
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\livemaker\lsb\lmscript.py", line 650, in get_menu_choices
    for line_no, menu in self.get_menus(run_order):
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\livemaker\lsb\lmscript.py", line 631, in get_menus
    menu = make_menu(self, i, pylm=self.pylm)
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\livemaker\lsb\menu.py", line 455, in make_menu
    return cls.from_lsb_command(lsb, index, **kwargs)
  File "c:\program files (x86)\microsoft visual studio\shared\python37_64\lib\site-packages\livemaker\lsb\menu.py", line 433, in from_lsb_command
    raise KeyError(f"No matching jump for menu choice {text}")
KeyError: 'No matching jump for menu choice None'
```

**Comment by pmrowla** [comment]
Thanks for the file upload, I'll look into this when I get the chance

**Comment by pmrowla** [comment]
This issue should be resolved in latest (v1.0.1) release
